src/main/java/org/opentripplanner/transit/raptor/rangeraptor/standard/stoparrivals/Stops.java

```diff
@@ -77,12 +60,17 @@ public int calculateMinNumberOfTransfers(int stop) {
         return unreachedMinNumberOfTransfers();
     }
 
-    void setAccessTime(int time, RaptorTransfer access) {
+    void setAccessTime(int time, RaptorTransfer access, boolean bestTime) {
         final int stop = access.stop();
-        if (stops[round()][stop] == null) {
-            stops[round()][stop] = new AccessStopArrivalState<>(time, access);
-        } else {
-            stops[round()][stop] = new AccessStopArrivalState<>(time, access, stops[round()][stop]);
+        var other = getOrCreateStopIndex(round(), stop);
```

Comment:
Maybe some better name for `other`?

Reply:
I changed this to "existingArrival". That is not 100%, since it is potentially more than on arrival, but a bit better.